src/Hazelcast.Net/Core/JavaUuidOrder.cs

```diff
@@ -14,6 +14,7 @@
 
 using System;
 using System.Runtime.InteropServices;
+using Hazelcast.Core;
```

Comment:
Is this needed?

Reply:
No, removed at https://github.com/hazelcast/hazelcast-csharp-client/pull/763/commits/d40907e6cdf75465a9527659bbb5ad40d8ea1987